orca/templates/teleport-agent-test/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:jammy
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=Etc/UTC
+
+RUN apt update && \
```

Comment:
We have a pattern in the docker repo on setting the DEBIAN_FRONTEND and TZ for the install, let's use that here as well. https://github.com/liferay/liferay-docker/blob/master/templates/base/Dockerfile#L6